docs/manual/troubleshooting.tex

```diff
@@ -822,6 +822,10 @@
 You might want to add an \<export CHECKERFRAMEWORK=...> line to your
 \<.bashrc> file.
 
+\textbf{Ensure Line Endings are Handled Properly}: Run
+\texttt{git config --global core.autocrlf true} in Git Bash to prevent line ending
+issues between Windows and Unix-based systems, or execute Git commands directly
+from the Ubuntu terminal.
```

Comment:
```suggestion
\textbf{Ensure line endings are handled properly}: Run
\code{git config --global core.autocrlf true} to prevent line ending issues between
Windows and Unix-based systems.
```

Reply:
The `Ubuntu terminal` part is too distribution specific. Maybe there should be a general note about how to build on Windows/Unix?